dask/tests/test_distributed.py

```diff
@@ -69,10 +68,15 @@ def test_persist_nested(c):
     assert res[2:] == (4, [5])
 
 
-@pytest.mark.skipif(_numpy_120, reason="https://github.com/dask/dask/issues/7170")
 def test_futures_to_delayed_dataframe(c):
     pd = pytest.importorskip("pandas")
     dd = pytest.importorskip("dask.dataframe")
+
+    from dask.array.numpy_compat import _numpy_120
```

Comment:
I don't think that will work, because the import already fails if dask.array is not present.

Reply:
... or when dask.array.numpy_compat is importing numpy right at the top.

Reply:
right, but we only care about the numpy version if dask.array _is_ present

Reply:
But the test fails before it can skip if it is *not* present. You have to wrap that line into `importorskip` too.

Reply:
Oh, because dask.dataframe needs dask.array, this is automatically fulfilled. Nevermind then.